x/farming/keeper/reward.go

```diff
@@ -226,14 +235,6 @@ func (k Keeper) Harvest(ctx sdk.Context, farmerAcc sdk.AccAddress, stakingCoinDe
 		totalRewards = totalRewards.Add(rewards...)
 	}
 
-	ctx.EventManager().EmitEvents(sdk.Events{
-		sdk.NewEvent(
-			types.EventTypeHarvest,
-			sdk.NewAttribute(types.AttributeKeyFarmer, farmerAcc.String()),
-			sdk.NewAttribute(types.AttributeKeyRewardCoins, totalRewards.String()),
-		),
-	})
-
```

Comment:
These events emitting should be left for some user could be checking `MsgHarvest` is normally committed